robot_skills/src/robot_skills/arm/grasp_sensor.py

```diff
@@ -87,20 +91,18 @@ class GraspSensor(RobotPart):
     """
     Sensor to detect whether or not the robot is holding an object.
     """
-
-    def __init__(self, robot_name, tf_listener, side):
+    def __init__(self, robot_name, tf_listener, sensor_topic):
         """
         constructor
 
         :param robot_name: robot_name
         :param tf_listener: tf_server.TFClient()
-        :param side: left or right arm of the robot.
+        :param sensor_topic: name of the topic where measurements are published
         """
         super(GraspSensor, self).__init__(robot_name=robot_name, tf_listener=tf_listener)
         # Init grasp sensor subscriber
         self._grasp_sensor_state = GripperMeasurement(0.0)
-        rospy.Subscriber("/" + robot_name + "/" + side + "_arm/proximity_sensor",
-                         std_msgs.msg.Float32MultiArray, self._grasp_sensor_callback)
+        rospy.Subscriber("/" + robot_name + "/" + sensor_topic, Float32MultiArray, self._grasp_sensor_callback)
```

Comment:
Now we force the robot name to be in the topic. Probably that is always the case, but I prefer we don't apply any unesesarry contraints on the topic. Let the user define the entire topic.

Reply:
Almost all topics within the interface are prefixed with the robots name. I think it would cause more trouble than its worth at this stage. There are limits to how generic we want to make these interfaces. And being able to quickly change the prefix is an important part of making an interface for a new robot.

Reply:
But I am pretty sure that other robot parts don't do this. So please keep it consistent.

Reply:
@PetervDooren I don't think we do this somewhere else. So therefore I still think we should change this.